.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
What does this accomplish for PHPC? Our only direct dependencies are the libmongoc and libmongocrypt submodules, and I doubt it's going to help us write Autoconf files :)

Would this be better suited to PHPLIB? I see that Composer is handled within [dependabot/dependabot-core](https://github.com/dependabot/dependabot-core).

Reply:
@jmikola FWIW, Dependabot could open pull requests for updates to submodules, but it currently can't handle tags and can only update to the latest commit.

I'm fine with having automatic updates to GitHub Actions dependencies so we don't have to manually keep track of them.

Reply:
Oh, my mistake for not realizing what `package-ecosystem: "github-actions"` did. SGTM.